src/Domain/Creators/CylindricalBinaryCompactObject.hpp

```diff
@@ -167,6 +167,12 @@ class CylindricalBinaryCompactObject : public DomainCreator<3> {
     static constexpr Options::String help = {
         "Initial refinement level. Applied to each dimension."};
   };
+  struct AdditionToOuterRadialRefinementLevel {
```

Comment:
(suggestion) We have a mechanism to support very flexible initial refinement in the input file now, like this:

```yaml
InitialRefinement:
  InnerRegion: [1, 2, 3]
  OuterRegion: [2, 3, 4]
# or:
InitialRefinement: [1, 2, 3]
# or:
InitialRefinement: 1
# or:
InitialRefinement:
  CACylinder: 0
  CAWedge: 1
  # ...
```

All of these are supported with a single option holding a `std::variant`, using the `domain::ExpandOverBlocks` utility and providing names for the blocks and groups of blocks. If you decide to adopt this, here's the commit that added this to the Cylinder domain creator: https://github.com/nilsleiffischer/spectre/commit/9907b68e96c95370dcae57f355ba099c303c364f